rules/library.bzl

```diff
@@ -985,6 +985,10 @@ def apple_library(
             tags = tags_manual,
             defines = defines + swift_defines,
             testonly = testonly,
+            always_include_developer_search_paths = select({
```

Comment:
I suppose we only need the rules_swift bump because we're applying this directly vs. in the swift_kwargs below.

Is it easy to not require the rules_swift bump? I don't really feel strongly either way just wanted to ask.

Reply:
That's a good callout, aligned with feedback I got from @luispadron too. I guess one looking to use the flag can simply use kwargs indeed. 

For those looking to use the flag It would simplify things if `rules_ios` was on this `rules_swift` version though. So i guess I can repurpose this PR to simply bump `rules_swift` if folks don't have any concerns with that.

Reply:
@mattrobmattrob Q: What would be the issue with the `rules_swift` bump here specifically? I could make the changes in a way that for us everything works. But isn't this a good opportunity to add support for this flag for all consumers?

Reply:
Mostly just that this requires a rules_swift bump for consumers. Not necessarily a bad thing though.